data/campaigns/The_Rise_Of_Wesnoth/units/Wesfolk_Leader.cfg

```diff
@@ -19,7 +19,7 @@
     {AMLA_DEFAULT}
     cost=60
     usage=mixed fighter
-    description= _ "Born in the Wesfolk aristocracy, she and her people were excluded from society by their lords, who betrayed their loyalty when a war against Haldric’s people was being lost. This outcast still conserves her nobleness in her veins and on the battlefield she has earned valuable experience, which continues increasing with time, as does her natural leadership."
+    description= _ "Born in the Wesfolk aristocracy, she and her people were excluded from society by their lords, who betrayed their loyalty when they started losing a war against Haldric's people. This outcast still conserves her nobleness in her veins, and on the battlefield she has earned valuable experience, which continues to increase with time, as does her natural leadership."
```

Comment:
I believe this is fine, but 'Haldric's people' needs to use a 'curly' apostrophe.

Reply:
I think I might prefer an infinitive rather than a gerund here as well - "when they started to lose".